bench/dashboard/style.css

```
@@ -1,510 +1,252 @@
-@import "https://fonts.googleapis.com/css2?family=Sora:wght@400;500;600;700;800&family=IBM+Plex+Mono:wght@400;500;600&display=swap";
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap'" (import-notation)

(import-notation)